packages/fastify/modules/plugin/plugin.js

```diff
@@ -0,0 +1,81 @@
+const fp = require('fastify-plugin');
+const merge = require('merge-options');
+const {
+  Prometheus,
+  createMetricTypes,
+  createRequestRecorder,
+  createGcObserver,
+  defaultNormalizers,
+  getSummary,
+  getContentType,
+} = require('@promster/metrics');
+
+const defaultRouteOptions = {
+  route: { method: 'GET', url: '/metrics' },
+};
+
+let recordRequest;
+let upMetric;
+
+const extractPath = req => req.raw.originalUrl || req.raw.url;
+const getRequestRecorder = () => recordRequest;
+const signalIsUp = () => upMetric && upMetric.set(1);
+const signalIsNotUp = () => upMetric && upMetric.set(0);
+
+const plugin = async function(fastify, options) {
```

Comment:
I think this should be `createPlugin` so we can pass in options as with other plugins. Like what metrics are wanted and what normalizers should be used?

Reply:
I agree with `createPlugin` naming
regarding options, I hope this would answer: https://github.com/tdeekens/promster/pull/169#discussion_r298829431